cosmwasm/deploy-contract.js

```diff
@@ -114,7 +114,7 @@ const programHandler = () => {
 
     program.addOption(
         new Option('-e, --env <env>', 'environment')
-            .choices(['local', 'devnet', 'stagenet', 'testnet', 'mainnet'])
+            .choices(['local', 'devnet', 'stagenet', 'testnet', 'devnet-sui', 'mainnet'])
```

Comment:
[Task](https://axelarnetwork.atlassian.net/browse/AXE-4659) to fix this properly